batch/batches/ExportCsv/Engine/KEntryExportEngine.php

```diff
@@ -13,7 +13,14 @@ protected function getFilterOrderBy()
 
 	protected function getItemList($filter, $pager)
 	{
-		return KBatchBase::$kClient->baseEntry->listAction($filter, $pager);
+		$clientTag = KBatchBase::$kClient->getClientTag();
+		if(is_object($filter->advancedSearch))
+		{
+			KBatchBase::$kClient->setClientTag($clientTag ." useESearch");
+		}
+		$items = KBatchBase::$kClient->baseEntry->listAction($filter, $pager);
```

Comment:
need to add try catch and if failed to reset the previous clientTag else we will be stuck with the modified one for the rest of the process